addons/mail/static/tests/mock_server/mock_models/discuss_channel.js

```diff
@@ -124,7 +124,7 @@ export class DiscussChannel extends models.ServerModel {
             if (partner.id === this.env.user.partner_id) {
                 continue; // adding 'yourself' to the conversation is handled below
             }
-            const body = `<div class="o_mail_notification">invited ${partner.name} to the channel</div>`;
+            const body = `<div class="o_mail_notification">${this.env.user.name} invited ${partner.name} to the channel</div>`;
```

Comment:
This is actually lacking the link and the proper formatting, the mock server should be adapted to changes in python.